Resources/Storage/GFAL2_SRM2Storage.py

```diff
@@ -205,3 +208,37 @@ def __getProtocols( self ):
       return S_ERROR( "GFAL2_SRM2Storage.__getProtocols: No local protocols defined and no defaults found." )
 
     return S_OK( protocolsList )
+
+  def getOccupancy(self):
```

Comment:
No no, there is no lcg_utils anymore. 
I will provide you with an implementation for that

Reply:
I am not sure what exactly you are expecting.

Is this ok ? 
```
In [14]: ctx.getxattr('srm://srm-eoslhcb.cern.ch', 'spacetoken.token?LHCb-EOS')
Out[14]: '{ "spacetoken": "LHCb-EOS", "owner": "lhcb", "totalsize": -1, "unusedsize": 429499145907438, "usedsize": 0, "guaranteedsize": 5000000000000000, "lifetimeleft": -1, "retention": "REPLICA", "accesslatency": "ONLINE" }'
```

Reply:
And what is `ctx`? Trying on lxplus:

```
In [47]: import ctx
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-47-56f9a9ccee81> in <module>()
----> 1 import ctx

ImportError: No module named ctx
```

Reply:
Oh is it just a context?

Reply:
yes. But as I said if you want I can implement it, I was just asking if these are the info you were looking for.

Reply:
Alright so let's do that this is merged, and then it's left to be correctly implemented. What's is added here right now (`lcg_util`) is what is used today to calculate the space left for space tokens. It works properly so I am not sure why you say that "there's no lcg_utils anymore". The call you posted seems showing the same result, but honestly I didn't properly check, and also it should be seen if the results are consistent among several type of SEs.

Reply:
`lcg_util` is gfal version 1, it is deprecated and should not be used, that's what I meant. 
OK, as soon as it is merged, I will implement the correct version